### PR TITLE
Remove proxy-protocol

### PR DESCRIPTION
**Pull request by stewartpark**
/cc @kruppel @kev71187 @jeremyd @sasso 

tl;dr: Our healthcheck has been sending us an empty string with proxy protocol header appended. So this PR is to disable proxy protocol on ELB since we are not utilizing it

To work around Kubernetes's S-NAT'd incoming node port traffic in order to preserve source IP, many production deployments use proxy protocol, but since we set up ELB right to be HTTP, this hasn't been our problem, since it'd set `X-Forwarded-For`.

Though we have enabled it by default, and it was causing our apps having lots of parse errors, since our apps don't understand how proxy protocol works.

The below is the packets we're getting as healthchecks to our apps:
```
PROXY TCP4 10.100.102.67 10.100.102.67 28225 32004
PROXY TCP4 10.100.101.66 10.100.101.66 63765 32004
PROXY TCP4 10.100.101.66 10.100.101.66 49552 32004
PROXY TCP4 10.100.100.36 10.100.100.36 17443 32004
PROXY TCP4 10.100.101.66 10.100.101.66 49928 32004
PROXY TCP4 10.100.100.36 10.100.100.36 45398 32004
PROXY TCP4 10.100.102.67 10.100.102.67 19289 32004
PROXY TCP4 10.100.102.67 10.100.102.67 20684 32004
PROXY TCP4 10.100.101.66 10.100.101.66 48946 32004
PROXY TCP4 10.100.101.66 10.100.101.66 28947 32004
PROXY TCP4 10.100.101.66 10.100.101.66 64861 32004
PROXY TCP4 10.100.103.186 10.100.103.186 48271 32004
PROXY TCP4 10.100.103.186 10.100.103.186 10005 32004
PROXY TCP4 10.100.103.186 10.100.103.186 42040 32004
PROXY TCP4 10.100.103.186 10.100.103.186 7869 32004
PROXY TCP4 10.100.103.186 10.100.103.186 8891 32004
...
```

And it's supposed to be empty strings for puma to ignore it.

### CODE REVIEW

**Comment by sasso** [comment]
Tested in one of our internal apps and parse error goes away!
